### PR TITLE
feat: add bun related docs

### PR DESCRIPTION
**Pull request by stonega**

### CODE REVIEW

**Comment by MasedMSD** [comment]
@KnorpelSenf The issue of adding Bun is becoming more and more pressing. At the moment even plugin conversation, which could throw errors before, is working more and more stable.

+1 from me for adding documentation for Bun

Btw, many users from Russian-speaking chat also actively use it.

**Comment by KnorpelSenf** [comment]
@winstxnhdw can you confirm that they have fixed their support for conversations, i18n, etc?

File: site/docs/guide/getting-started.md [hunk 1]
```diff
@@ -150,6 +150,89 @@ in your terminal before you execute `node bot.js`.
...
+cd my-bot
+
+# Run bun init to scaffold a new project, for this tutorial, just press enter to accept the default answer for each prompt.
+bun init
+
+# Install grammY.
+bun add grammy
+```
+
+Create a new empty text file, e.g. called `bot.ts`.
```
Comment (MasedMSD) [inline site/docs/guide/getting-started.md:173]: Aren't bun automatically creating index.ts if we `just press enter to accept the default answer for each prompt.`

**Comment by winstxnhdw** [comment]
> @winstxnhdw can you confirm that they have fixed their support for conversations, i18n, etc?

Was i18n ever an issue?

**Comment by MasedMSD** [comment]
> conversations

image

**Comment by KnorpelSenf** [comment]
I think all plugins compiled with dnt were, but I have never used bun, it's only what people reported

**Comment by KnorpelSenf** [comment]
So they finally fixed it? That's pretty good!

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
Please also mention bun in https://grammy.dev/guide/introduction#prerequisites-to-getting-started

**Comment by KnorpelSenf** [comment]
Actually, that section is pretty outdated anyway. Neither the part about Node nor the part about Deno are accurate anymore. I will fix things up in a subsequent PQ. We can keep this as-is for now.

**Comment by KnorpelSenf** [comment]
@stonega what about https://grammy.dev/guide/api#type-definitions-for-the-bot-api?

**Comment by rojvv** [comment]
Closing due to inactivity.

**Pull request closed without merging**